docs/UserGuide.md

```diff
@@ -664,7 +666,7 @@ Example:
     Attendance List is sorted by attendance status for Event: event1
     
 ### 4.5. Performance
```

Comment:
Regarding the `Performance` category, I'm not too sure if it would be better to change it to `Performance List` since everything under this heading is about `Performance List` instead of `Performance`

Reply:
valid point! will change that